js/languages/en-US.json

```diff
@@ -85,6 +85,41 @@
     "generalTab": "General",
     "pageTab": "Page",
     "addressesTab": "Addresses",
+    "advancedTab" : {
+      "_sectionName" : "Advanced",
+      "smtp" : {
+        "_sectionName" : "SMTP Integration",
+        "_testAction" : "Test Integration",
+        "notifications" : "Notifications",
+        "username" : "Username",
+        "password" : "Password",
```

Comment:
Our linter does not lint JSON, so it's kind of up to us to be consistent. Could you not have a space before the colon?